app/src/main/java/com/example/mbanking/pages/AllTransactions.kt

```diff
@@ -36,14 +36,30 @@ import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
 import com.example.mbanking.R
+import com.example.mbanking.details.DateBottomSheet
 import com.example.mbanking.details.TransactionCard
+import com.example.mbanking.util.sortByDate
+
+
+var newStartDate = "01.01.1970"
+var newEndDate = "30.01.3000"
```

Comment:
in file with composable functions should be composable functions only. I recommend move this variables into another util-file

Reply:
or you won't need this vars when rewrite your code to lambdas to get filter results

Reply:
done